srcpkgs/firewalld/template

```diff
@@ -0,0 +1,54 @@
+# Template file for 'firewalld'
+pkgname=firewalld
+version=2.2.1
+revision=1
+build_style=gnu-configure
+hostmakedepends="autoconf pkg-config docbook-xsl xmlcatmgr intltool libxslt
+ python3 docbook-xsl glib-devel"
+depends="python3-gobject python3-dbus nftables ipset polkit"
+short_desc="Dynamically managed firewall with support for network zones"
+maintainer="Daniel Martinez <danielmartinez@cock.li>"
+license="GPL-2.0-or-later"
+homepage="https://firewalld.org/"
+distfiles="https://github.com/firewalld/firewalld/releases/download/v${version}/firewalld-${version}.tar.bz2"
+checksum=5215ba30236ee1e3df2c2292465a9ff605b9c445dcab2e37da4961cb27c7f36e
+python_version=3.12
+conf_files="/etc/firewalld/*"
+# firewalld does not believe it is root in the chroot and fails
+make_check=no
+
+post_install() {
+	mv ${DESTDIR}/etc/modprobe.d ${DESTDIR}/usr/lib
+	vsv ${pkgname}
+}
+
+firewalld-applet_package() {
+	short_desc+=" - desktop applet"
+	depends="firewalld>=${version}_${revision} python3-pyqt6-gui
+	 python3-pyqt6-dbus python3-pyqt6-widgets libnotify"
+	pkg_install() {
+		vmove etc/xdg/autostart/firewall-applet.desktop
+		vmove usr/bin/firewall-applet
+		for size in 16 22 24 32 48; do
+	 		vmove "usr/share/icons/hicolor/${size}x${size}/apps/firewall-applet*"
+		done
+		vmove "usr/share/icons/hicolor/scalable/apps/firewall-applet*"
+		vmove usr/share/man/man1/firewall-applet.1
+	}
+}
+
+firewalld-config_package() {
```

Comment:
the binary is `/usr/bin/firewall-config`, maybe the subpackage should be named `firewall-config` ?

Reply:
same thing, config for firewalld not any firewall